agent/local/state_test.go

```diff
@@ -1916,8 +1916,15 @@ func checksInSync(state *local.State, wantChecks int) error {
 func TestState_Notify(t *testing.T) {
 	t.Parallel()
 
+	consulLogger := hclog.New(&hclog.LoggerOptions{
```

Comment:
This should probably explicitly set the Output at least during this massive transition so everything is crystal clear, even if Stderr is the default.